project-base/app/src/Controller/Front/SocialNetworkController.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Front;
```

Comment:
move this controller into the framework package, please. Beware the injected classes then, so they are not from project-base